packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -113,37 +112,14 @@ export default class SalesforceCRMService implements CRM {
     });
   };
 
-  private salesforceContactCreate = async (attendees: Person[]) => {
+  private getSalesforceUserFromEmail = async (email: string) => {
     const conn = await this.conn;
-    const createdContacts = await Promise.all(
-      attendees.map(async (attendee) => {
-        const [FirstName, LastName] = attendee.name ? attendee.name.split(" ") : [attendee.email, ""];
-        return await conn
-          .sobject("Contact")
-          .create({
-            FirstName,
-            ...(LastName ? { LastName } : {}),
-            Email: attendee.email,
-          })
-          .then((result) => {
-            if (result.success) {
-              return { Id: result.id, Email: attendee.email };
-            }
-          });
-      })
-    );
-    return createdContacts.filter(
-      (contact): contact is Omit<ContactSearchResult, "attributes"> => contact !== undefined
-    );
+    return await conn.query(`SELECT Id, Email FROM User WHERE Email = '${email}'`);
   };
 
-  private salesforceContactSearch = async (event: CalendarEvent) => {
+  private getSalesforceUserFromOwnerId = async (ownerId: string) => {
```

Comment:
When querying contacts, it only returns the `ownerId`. So need to make another query to Salesfoce for the user email.